test/Microsoft.TestPlatform.Common.UnitTests/ExtensionFramework/TestPluginCacheTests.cs

```diff
@@ -459,6 +459,11 @@ protected override IEnumerable<string> GetFilteredExtensions(List<string> extens
             this.Action?.Invoke();
             return extensions;
         }
+
+        new public void SetupAssemblyResolver(string extensionAssembly)
+        {
+            base.SetupAssemblyResolver(extensionAssembly);
```

Comment:
Do we need this?

Reply:
Now that we don't pass Coverlet adapter path to host "before test run" to avoid assemblies resolution error inside inproc constructor we need to add coverlet folder to assembly resolver. In production is not a problem because assembly resolver is created thanks to TestPluginCache discover methods, but for testing purpose we need a way to create an assembly resolver, but inproc constructor doesn't need to call discover methods but only to add custom resolution folder. It's needed for testing purpose.

Reply:
I mean, it looks like we are just calling base in here...

Reply:
Yep, `SetupAssemblyResolver` was private and not accessible outside `TestPluginCacheTests`, but I need to call it before pass to InProc collector for testing https://github.com/microsoft/vstest/pull/2288/files#diff-ebb20c6bd7cf65eed5a70867b1e73818R100 

So I've updated visiblity from private to protected and exposed to tests. I need a `new` because I kept same method name hiding base method. Would you like to expose `SetupAssemblyResolver` public?

Reply:
Nope, my bad - missed that it was protected. Thanks.